text-editor/text-editor-002-en.md

```diff
@@ -92,75 +95,78 @@ was succesfully embed into an other node when we wanted to make it bold :
 }
 ```
 
-However, things were getting way more complicated when we wanted to update
-some selected text that were starting in a paragraph and ending on an other,
-or also when the selection was overlapsing on different tags, such as
+However, things were getting way more complicated from the moment we worked on
+more edgy cases, i.e. update some text that was starting in a paragraph and
+ending in an other. Selection overlapsing on different and nested tags, such as
 
 ```
 <b>Hello <em>John Doe</em></b>, how are you ?
     |---------------|
 selection start   selection end
 ```
 
-We realized that applying such changes on the AST would have beed difficult
+was a daunting too. Applying such changes on the AST would have been difficult
 for us. In the mean time, the developer who started to work on the AST was
-required to work on an other project so we decided not to use this AST anymore
-as it would have been very time consuming to develop and maintain (and also
-the learning surve was not easy for newcomers)
+required on an other project, so we decided not to use this AST anymore. Because
+it was the corner stone of the backend application, keeping it at that level of
+complexity and abstraction was too risky, time consuming to develop and maintain,
+and the learning curve was giving nightmares to newcomers.
```

Comment:
Should we also point out the FP style used for this AST that we were not comfortable with ?